vendor/cloud.google.com/go/storage/bucket.go

```diff
@@ -1977,7 +1977,7 @@ func (it *ObjectIterator) fetch(pageSize int, pageToken string) (string, error)
 	if projection == ProjectionDefault {
 		projection = ProjectionFull
 	}
-	req.Projection(projection.String())
+	req.Projection("noAcl")
```

Comment:
For this I think we should debug as to why we are not getting the right value from the projection.string when we set it to noAcl in gcsfuse code.

Reply:
done.

Reply:
Checked by performing manual listing operation and it's working correctly now.

Reply:
Please delete the lines 1976-1979 (which use the projection variable) and add a comment that this is done to ask for higher number of results.